modules/apps/portal-security/portal-security-script-management-web/src/main/resources/META-INF/resources/js/components/ScriptManagementContainer.tsx

```diff
@@ -76,6 +78,10 @@ export default function ScriptManagementContainer({
 					: 'danger',
 			});
 
+			if (editScriptManagementConfigurationResponse.ok) {
+				setTimeout(() => window.location.reload(), 1000);
+			}
```

Comment:
Hey @danvb112, could we use a state to validate the alert, thus avoiding the need to reload the page?

Reply:
Done! Thanks @igor-franca